src/Components/Actions/ActionsWindow.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import GeneralWindow from 'Components/GeneralComponents/GeneralWindow';
+
+export default function ActionsWindow({ unmountActionsWindow }) {
+    return (
+        <GeneralWindow unmountGeneralWindow={unmountActionsWindow}>
+        <br />
```

Comment:
please fix indentation.
also - please check out the formatting standard I sent at Discord. (let me know if it works for you)